src/scss/components/_list.scss

```diff
@@ -100,4 +109,97 @@
       padding-right: inherit;
     }
   }
+
+  &__title {
+    color: get-color(rastaGray);
+    display: block;
+    font-size: 12px;
+    font-weight: get-weight(bold);
+    text-align: left;
+    text-transform: uppercase;
+  }
+
+  &--on-column {
+    &[class*='col-'] {
+      display: inline-block;
+      margin-left: 0;
+
+      + & {
+        margin-left: auto;
+        margin-right: 0;
+      }
+
+      .list__item {
+        width: 100%;
+      }
+    }
+  }
+
+  &--social {
+    display: flex;
+    flex-grow: 1;
+
+    .list__item {
+      border-radius: 50%;
+      height: 48px;
+      margin-left: 6%;
+      text-indent: -9999px;
+      width: 48px;
+
+      a {
+        padding: 0;
+        width: 100%;
+      }
+
+      img {
+        display: block;
+        width: 100%;
+      }
+
+      &:first-child {
+        margin-left: 0;
+      }
+    }
+  }
+
+  &--app {
+    display: flex;
+    flex-direction: row;
+    flex-grow: 1;
+
+    .list {
+      &__item {
+        border-radius: 4px;
+        flex-grow: 1;
+        margin: 0;
+        width: 100%;
+
+        a {
+          display: block;
+        }
+
+        &:last-child {
+          margin-left: 5px;
+
+          @include media(normal) {
+            margin-left: 0;
+            margin-top: get-space(element, small);
+          }
+        }
+      }
+
+      &__image {
+        width: 100%;
+
+        @include media(small) {
+          max-width: initial;
```

Comment:
`max-width: initial` can be moved to `&__image` scope? Because there is not a `max-width` on parent element

Reply:
Done! 👍